guides/intro_to_git.md

```diff
@@ -0,0 +1,316 @@
+# Overview
+This page is intended as a starting point for R users who are new to using Git. It shows you the basic commands and the workflow to use Git on your own, in conjunction with using RStudio desktop or RStudio Cloud. 
+
+To use Git as a team you should complete this section and the following section on [using git collaboratively](02_using-git-collaboratively.md).
+
+## What is version control?
+Version control is the practice of tracking and managing any changes on project's code, files or folders over time. This allows you to observe a detailed history of the changes made and enables your team members to collaborate on the same project.
```

Comment:
Not clear what Git is here, esp. considering the file is called intro_to_git

Reply:
I think we need a ticket to update the introduction_to_git over to RAP COP as well